jib-core/src/main/java/com/google/cloud/tools/jib/http/FailoverHttpClient.java

```diff
@@ -128,12 +128,12 @@ private static HttpTransport getInsecureHttpTransport() {
   private final Consumer<LogEvent> logger;
   private final Supplier<HttpTransport> secureHttpTransportFactory;
   private final Supplier<HttpTransport> insecureHttpTransportFactory;
-  private final boolean retryOnIoException;
 
   private final ConcurrentHashMap<String, Failover> failoverHistory = new ConcurrentHashMap<>();
 
   private final Deque<HttpTransport> transportsCreated = new ArrayDeque<>();
   private final Deque<Response> responsesCreated = new ArrayDeque<>();
+  private final boolean turnOffRetry;
```

Comment:
This is now like a global and ultimate turn-off switch (only there for making some tests work).

Reply:
qq: I could be missing some context here. What if we make this `turnOnRetry` instead?

Reply:
So my thinking for `turnOff` and some context: if we didn't have tests, and we wouldn't have this field. This PR also introduces another variable (`retryOnIoException`) in the method below, so it could a bit be confusing if we have both "turn on retry" and "retry on something", but maybe that's only me. But more relevantly, I thought `turnOffRetry` sort of implies that retry is on by default (which is what we want and will always be the case if it were not for tests), and if you don't want retrying for some reason, then you have to explicitly turn it off. But if people with a fresh eye without the context may think this is more confusing, I can just flip it.

Reply:
I see! Your logic of having the retry on by default makes sense but I usually find it a little hard to parse negative booleans because the meaning of `true` or `false` for such variables gets inverted. Or perhaps we could name this `enableRetry`? `enableRetry` gives it more global sense and `retryOnIOException` is specific to a particular case.

Reply:
Sounds good. Done.